Marlin/src/core/serial.h

```diff
@@ -144,7 +144,7 @@ extern uint8_t marlin_debug_flags;
 #define SERIAL_ECHOLNPAIR(V...) _SELP_N(NUM_ARGS(V),V)
 
 #define SERIAL_ECHOPGM(S)           (serialprintPGM(PSTR(S)))
-#define SERIAL_ECHOLNPGM(S)         (serialprintPGM(PSTR(S "\n")))
+#define SERIAL_ECHOLNPGM(S)         do{ SERIAL_ECHOPGM(S); SERIAL_EOL(); }while(0)
```

Comment:
This shouldn't be needed, because serial output doesn't use translation strings. Also, the original change concatenating `"\n"` in this macro was a big saver of PROGMEM. If this change is needed to fix an ExtUI issue, then I suggest that the changed version be conditional on extensible UI.